apps/opik-backend/src/main/java/com/comet/opik/infrastructure/OpikConfiguration.java

```diff
@@ -37,4 +37,8 @@ public class OpikConfiguration extends Configuration {
     @Valid
     @NotNull @JsonProperty
     private RateLimitConfig rateLimit = new RateLimitConfig();
+
+    @Valid
+    @NotNull @JsonProperty
+    private OpikMetadataConfig metadata = new OpikMetadataConfig();
```

Comment:
Minor: everything is Opik in this service and repo, you don't need the Opik prefix in this object name.